### PR TITLE
Fixing some minor inconsistencies

### PR DESCRIPTION
**Pull request by oscrx**
Title says it all.

Love the project btw!

### CODE REVIEW

File: roadmap/BUTTERCUP_BROWSER_EXTENSION.md [hunk 1]
```diff
@@ -9,17 +9,17 @@ The browser extension allows users to connect their archives in their browser fo
...
- * Support for Opera
  * Autofilling of forms
  * Password generation
 
 **Planned features:**
- * Loading from the filesystem
  * Integration with the [my.buttercup](OVERALL.md#mybuttercup) service
  * Password renewal/update
  * Support for Safari
+ * Support for Opera
```
Comment (perry-mitchell) [inline roadmap/BUTTERCUP_BROWSER_EXTENSION.md]: This can be removed - we won't be supporting Opera again.

**Review by perry-mitchell — CHANGES_REQUESTED** [review]
1 minor change.. Looks great, thanks!

**Comment by oscrx** [comment]
All fixed now :)

**Review by perry-mitchell — APPROVED** [review]

**Comment by perry-mitchell** [comment]
@oscrx Thanks!

**Pull request merged**